library/core/src/mem/maybe_uninit.rs

```diff
@@ -851,4 +851,124 @@ impl<T> MaybeUninit<T> {
     pub fn slice_as_mut_ptr(this: &mut [MaybeUninit<T>]) -> *mut T {
         this.as_mut_ptr() as *mut T
     }
+
+    /// Copies the elements from `src` to `this`, returning a mutable reference to the now initalized contents of `this`.
+    ///
+    /// If `T` does not implement `Copy`, use [`clone_from_slice`]
+    ///
+    /// This is equivalent to [`slice::copy_from_slice`].
+    ///
+    /// # Panics
+    ///
+    /// This function will panic if the two slices have different lengths.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(maybe_uninit_write_slice)]
+    /// use core::mem::MaybeUninit;
+    ///
+    /// let mut dst = [MaybeUninit::uninit(); 32];
+    /// let src = [0; 32];
+    ///
+    /// let init = MaybeUninit::copy_from_slice(&mut dst, &src);
+    ///
+    /// assert_eq!(init, src);
+    /// ```
+    ///
+    /// ```
+    /// #![feature(maybe_uninit_write_slice, vec_spare_capacity)]
+    /// use core::mem::MaybeUninit;
```

Comment:
```suggestion
    /// use std::mem::MaybeUninit;
```

Even in the docs for `core` it's more common to use `std` because it's what most people use and the terminology is more helpful for people coming from other languages. 😄 

Note: I have not put a comment everywhere this is done

Reply:
I checked, there are only 12 instances of docs using `//[/!] use core`, I'll probably open a PR to change those too. Consistency is key here. 😄

```
rg '//[/!] use core' --count
library/core/src/future/ready.rs:1
library/core/src/future/pending.rs:1
library/core/src/future/poll_fn.rs:2
library/core/src/task/ready.rs:3
library/core/src/macros/mod.rs:1
library/core/src/fmt/mod.rs:1
library/core/src/panic.rs:1
library/std/src/primitive_docs.rs:2
src/tools/cargo/src/cargo/core/features.rs:1
```

Reply:
fixed these